dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -231,7 +231,8 @@
     <x:Double x:Key="PaneToggleButtonWidth">40</x:Double>
     <x:Double x:Key="NavigationViewCompactPaneLength">40</x:Double>
     <x:Double x:Key="NavigationViewTopPaneHeight">40</x:Double>
-   
+    <x:Double x:Key="NavigationViewTopModeContentPaneMinWidth">48</x:Double>
```

Comment:
We should strive to be consistent as best as possible with our theme resource naming here. As far as I can see, only the theme resource `NavigationViewTopPaneHeight` uses a slightly different prefix for the specific NavigationView theme resources in top pane display mode. The overwhelming majority uses the prefix "TopNavigationView" so I suggest we should use that prefix here as well. "NavigationViewTopMode" is yet another new prefix here and we should avoid that.

(Out of scope here but mentioned for completeness: For consistency, we should also add a `TopNavigationViewPaneHeight` theme resource later when, for example, the XAML resource lookup system supports a level of indirection to avoid a breaking change.)

The naming "NavigationViewTopModeContentPaneMinWidth" also seems to suggest to me that the NavigationView has a *content pane* with its min width configurable here. As that is not the case (there is only one pane), we should be careful how we name it here. The ContentControl placed inside this grid column is named "PaneCustomContentOnTopPane" so perhaps we could name it `TopNavigationViewPaneCustomContentMinWidth`?

Reply:
Very good point. I like "TopNavigationViewPaneCustomContentMinWidth", so I've updated the code now to use that name. Thanks for the suggestion!

> (Out of scope here but mentioned for completeness: For consistency, we should also add a TopNavigationViewPaneHeight theme resource later when, for example, the XAML resource lookup system supports a level of indirection to avoid a breaking change.)

Should there be/is there a tracking issue for that?

Reply:
I do plan to create a tracking issue for the XAML resource lookup system soon (I'm aware I'm already a few days late here 😅) but there is none for ensuring constistency for the *existing* NavigationView theme resources (like `NavigationViewTopPaneHeight` vs `TopNavigationViewPaneHeight`). We should create one (as otherwise this can fall off the radar pretty quickly). Feel free to create a tracking issue for this if you agree, @chingucoding 🙂

Reply:
Created #2906 for the navview resource naming scheme.

Reply:
Agree that we should try to be as consistent as possible, but it looks like we already have some inconsistencies in the naming and we have to be careful about changing the naming since that is potentially a breaking change. Lets try to follow the pattern as much as possible. @YuliKl  and @anawishnoff as FYI.